course/src/main/java/in/testpress/course/util/ExoPlayerUtil.java

```diff
@@ -271,6 +271,44 @@ public void onClick(View v) {
         });
     }
 
+    public void initializePlayer() {
```

Comment:
Check the level of abstraction. initializePlayer method should only have other method calls. It should not have details directly in initializePlayer.